go.mod

```diff
@@ -0,0 +1,238 @@
+module k8s.io/kubernetes
```

Comment:
erm, since when do we ship a go.mod? this broke some of my tooling which expects to find k8s in $GOPATH as a go package in CI / on dev machines 🙃 

I can change [my tool](https://sigs.k8s.io/kind), but this seems like an unrelated change to the intent of the PR?

Reply:
Yeah, that should be dropped. Only reason that was in the commit was because a find/replace found the dependency line mentioned in it locally

Reply:
fixup in https://github.com/kubernetes/kubernetes/pull/72963

Reply:
thank you!